packages/carbon-web-components/src/components/button/button.stories.ts

```diff
@@ -0,0 +1,429 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+// Below path will be there when an application installs `carbon-web-components` package.
+// In our dev env, we auto-generate the file and re-map below path to to point to the generated file.
+// @ts-ignore
+import Add16 from '@carbon/web-components/es/icons/add/16';
+import {
+  BUTTON_KIND,
+  BUTTON_TYPE,
+  BUTTON_SIZE,
+  BUTTON_TOOLTIP_ALIGNMENT,
+  BUTTON_TOOLTIP_POSITION,
+} from './button';
+import './index';
+import storyDocs from './button.mdx';
+
+const kind = {
+  [`Primary button (${BUTTON_KIND.PRIMARY})`]: BUTTON_KIND.PRIMARY,
+  [`Secondary button (${BUTTON_KIND.SECONDARY})`]: BUTTON_KIND.SECONDARY,
+  [`Tertiary button (${BUTTON_KIND.TERTIARY})`]: BUTTON_KIND.TERTIARY,
+  [`Danger button (${BUTTON_KIND.DANGER})`]: BUTTON_KIND.DANGER,
+  [`Danger tertiary button (${BUTTON_KIND.DANGER_TERTIARY})`]:
+    BUTTON_KIND.DANGER_TERTIARY,
+  [`Danger ghost button (${BUTTON_KIND.DANGER_GHOST})`]:
+    BUTTON_KIND.DANGER_GHOST,
+  [`Ghost button (${BUTTON_KIND.GHOST})`]: BUTTON_KIND.GHOST,
+};
+
+const types = {
+  [`Button`]: BUTTON_TYPE.BUTTON,
+  [`Reset`]: BUTTON_TYPE.RESET,
+  [`Submit`]: BUTTON_TYPE.SUBMIT,
+};
+
+const alignmentOptions = {
+  ['Start']: BUTTON_TOOLTIP_ALIGNMENT.START,
+  ['Center']: BUTTON_TOOLTIP_ALIGNMENT.CENTER,
+  ['End']: BUTTON_TOOLTIP_ALIGNMENT.END,
+};
+
+const positionOptions = {
+  ['Top']: BUTTON_TOOLTIP_POSITION.TOP,
+  ['Right']: BUTTON_TOOLTIP_POSITION.RIGHT,
+  ['Bottom']: BUTTON_TOOLTIP_POSITION.BOTTOM,
+  ['Left']: BUTTON_TOOLTIP_POSITION.LEFT,
+};
+
+const sizes = {
+  [`Small size (${BUTTON_SIZE.SMALL})`]: BUTTON_SIZE.SMALL,
+  [`Medium size (${BUTTON_SIZE.MEDIUM})`]: BUTTON_SIZE.MEDIUM,
+  [`Large size (${BUTTON_SIZE.LARGE})`]: BUTTON_SIZE.LARGE,
+  [`XL size (${BUTTON_SIZE.EXTRA_LARGE})`]: BUTTON_SIZE.EXTRA_LARGE,
+  [`2XL size (${BUTTON_SIZE.EXTRA_EXTRA_LARGE})`]:
+    BUTTON_SIZE.EXTRA_EXTRA_LARGE,
+};
+
+const defaultArgs = {
+  kind: BUTTON_KIND.PRIMARY,
+  tooltipAlignment: BUTTON_TOOLTIP_ALIGNMENT.CENTER,
+  tooltipPosition: BUTTON_TOOLTIP_POSITION.TOP,
+};
+
+const controls = {
+  buttonClassName: {
```

Comment:
Noticing that this is a new control over what we had. Guessing this is again to get us close to React Storybook. I might have missed this, but, are we aiming for 100% parity in this exercise? What's included / not included in that outside of whats already noted in the [migration guide](https://gist.github.com/IgnacioBecerra/0cba18e31c978e5632a9cbd8045e82e2)?

Reply:
`ButtonClassName` control is to keep parity with React, but the property already existed. Through this exercise, I don't believe we should introduce any new properties, but just try to align what's currently available to the react controls.

Reply:
That makes sense, also spoke with Kenny this morning at office hours on this. Will look out for these types of things. Thanks!